### ISSUE TITLE
Error on traversing template with Vue 2 filter syntax

### ISSUE THREAD

**Issue by adrienWeiss** [UnrefinedBrain/vue-metamorph#2]
Hello 👋 

first of all thank you for this tool, it comes at the exact time I'm trying to figure out a way to update calls to a lib I'm replacing in a very, very large codebase.
I'm still struggling (a lot), I guess because I am absolutely not familiar with AST parsing & manipulation, but I'm trying to go step by step with simple use cases first.
However I do have errors raised when traversing vue templates which use vue 2 filters syntax ( `{{ myVar | someFilter }}` )
It looks like you do have a type for this, `VFilter`, but I don't know if I am missing something in my conf or not. I guess it involves 
```js
parserOptions: {
    vueFeatures: {
      filter: true,
    },
}
```
somehow ?

the error I get is this 
```
[
  {
    filename: '<PATHTOMYFILE>.vue',
    error: TypeError: Cannot read properties of undefined (reading 'shift')
        at parseFilter (<PATHTOMYPROJECT>/node_modules/vue-eslint-parser/index.js:1088:24)
        at parseExpression (<PATHTOMYPROJECT>/node_modules/vue-eslint-parser/index.js:1250:22)
        at processMustache (<PATHTOMYPROJECT>/node_modules/vue-eslint-parser/index.js:2124:21)
        at <PATHTOMYPROJECT>/node_modules/vue-eslint-parser/index.js:3009:13
        at Parser.parse (<PATHTOMYPROJECT>/node_modules/vue-eslint-parser/index.js:2708:13)
        at parseAsSFC (<PATHTOMYPROJECT>/node_modules/vue-eslint-parser/index.js:6413:63)
        at parseForESLint (<PATHTOMYPROJECT>/node_modules/vue-eslint-parser/index.js:6398:53)
        at Module.parse (<PATHTOMYPROJECT>/node_modules/vue-eslint-parser/index.js:6406:12)
        at parseVue (file://<PATHTOMYPROJECT>/node_modules/vue-metamorph/dist/main.js:651:28)
        at transformVueFile (file://<PATHTOMYPROJECT>/node_modules/vue-metamorph/dist/main.js:685:33)
  }
]
```

Is it working for you ? do you see why it would not on my end ?

again, thanks for your help 👍

**Comment by UnrefinedBrain** [comment]
I managed to repro this: it throws this error when a filter has arguments.

I think I managed to fix it - can you bump to newest version 1.0.3 and see if that fixes it for you?

**Comment by adrienWeiss** [comment]
@UnrefinedBrain 👍 that fixed it, thank you very much 
closing this issue now, I'm investigating other errors I have running the traverser on my codebase and will open new issues accordingly
